README.txt

```diff
@@ -24,17 +27,15 @@ Benefits
 Users browse, search, and submit entries into the knowledge base. Entries can disprove, support, replicate, or peer-review other entries. The website calculates and informs users the status of an entry's validity using this system. An entry is most meritable if it follows the scientific method correctly. An entry is least meritable if it is just an opinion, or comment, and is less visible when sorting. Lower merit entries should not be dismissed, however, because they often inspire more meritable entries to be submitted.
 
 
-Mission Statement
+Mission Statement:
+To teach and engage 7+ billion people in investigative science and critical thinking. To maximize collaboration and assist humanity in reaching its collective intellectual potential.
 
-To engage everyone in problem-solving, using the scientific method as the most meritable method. To ensure that, through Methodocracy, humanity always has the ability to problem solve as much and as fast as possible.
 
+Vision Statement:
+To be known as the first place where thinkers and problem-solvers go. To ensure the survival of humanity, life, and consciousness. To alleviate suffering and improve the quality of life for sentient beings now and in the future.
 
-Vision Statement
 
-To be known as the first place where thinkers and problem-solvers go. To ensure the survival of humanity, life, and consciousness. To take care of suffering and improve the quality of life for everyone now and in the future.
-
-
-There is something missing on the Internet, its true potential has yet to be unlocked.
```

Comment:
Yes, I know. That's why I expanded it in the above paragraph into a longer sell.

Again, I think your target audience is going to want more meat from the pitches and flair. Something short and generic may just escape their attention. These people spend endless hours reading and writing. They're not going to get excited about a feel good statement without little content.